### ISSUE TITLE
Show current total stardust

### ISSUE THREAD

**Issue by javajohnHub** [PokemonGoF/PokemonGo-Bot#5500]
### Short Description

Show current total stardust a player currently has. Console output/telegram/discord

### Possible solution

I am not quite sure.

### How it would help others

I just think it would be nice to have that stat available somewhere.

**Comment by davidspeijer** [comment]
You see it in the stat overview when you login (only once).
